cpp/src/hash/hashing.cu

```diff
@@ -238,11 +405,15 @@ hash_partition_table(table_view const& input,
   //          {block0 partition(num_partitions-1) size, block1 partition(num_partitions -1) size, ...} }
   auto block_partition_sizes = rmm::device_vector<size_type>(grid_size * num_partitions);
 
+  auto scanned_block_partition_sizes = rmm::device_vector<size_type>(grid_size * num_partitions);
+
   // Holds the total number of rows in each partition
   auto global_partition_sizes = rmm::device_vector<size_type>(num_partitions, size_type{0});
 
+  auto row_partition_offset = rmm::device_vector<size_type>(num_rows);
+
   auto const device_input = table_device_view::create(table_to_hash, stream);
-  auto const hasher = experimental::row_hasher<MurmurHash3_32, has_nulls>(*device_input);
+  auto const hasher = experimental::row_hasher<MurmurHash3_32, hash_nulls>(*device_input);
```

Comment:
I was trying to make it more clear that this parameter meant specifically nulls in `table_to_hash` rather than nulls in the input table, since the former can be a subset of the latter. Hm, maybe I should add a comment there.

Reply:
Renamed to `hash_has_nulls`